src/de.rs

```diff
@@ -2015,6 +2015,21 @@ impl Error {
         self.inner.line.map(|line| (line, self.inner.col))
     }
 
+    /// Get the message associated with this error
+    pub fn message(&self) -> &str {
+        &self.inner.message
+    }
+
+    /// Get the keys associated with this error
+    pub fn key(&self) -> &[String] {
+        &self.inner.key
+    }
```

Comment:
This method also seems a little off, because it can return an empty slice in many situations.  I'm almost thinking it would be cleaner to return an option?

Reply:
Sure if you prefer it this way 😃 . I don't see the issue with an empty slice though. I would expect `Some(&[])` to have a different meaning from `None` when dealing with `Option<&[T]>`

Reply:
Yea, I think it would be better to be more explicit about the semantics here.